### ISSUE TITLE
'Figure' object has no attribute 'register_cmap' error when running volcano.csv

### ISSUE THREAD

**Issue by mdpoleto** [arielf/scripts#1]
Hi,

After downloading the full package, i tried to run xyz on volcano.csv but got the following error:

Traceback (most recent call last):
  File "xyz", line 515, in <module>
    sys.exit(main())
  File "xyz", line 510, in main
    xyz(params)
  File "xyz", line 256, in xyz
    fig.register_cmap(name='ariel', data=ariel_cdict)
AttributeError: 'Figure' object has no attribute 'register_cmap'

Any ideas on how to solve it?

**Comment by arielf** [comment]
Hi @marcelodepolo 

Thank you for the report, and taking the time to file the issue, and sorry for the trouble.

This looks like an API incompatibility between python library versions since it works fine for me on Ubuntu 16.04 and 18.04 with python 2.7 and the default `matplotlib`.

A quick way to solve it would be to edit your copy of the `xyz` script:

1) Comment out the two lines which call `register_cmap` in the source file (two lines) by placing a `#` char at the start of these two lines, like this:
```
# plt.register_cmap(name='ariel', data=ariel_cdict)
# plt.register_cmap(name='a', data=ariel_cdict)   # short alias
```

2) Change the line which uses the `ariel` custom color-map, to use some standard color-map like `jet`. IOW: rewrite this line: 
```
     cmapstr = myget(kw, '(?:col|cm)', 'ariel')
```
Into:
```
     cmapstr = myget(kw, '(?:col|cm)', 'jet')
```

I'll try to fix this soon in the repo.
I hope the suggested workaround works for you.
Thanks again for reporting the problem.

**Comment by marcelodepolo** [comment]
Thanks Ariel! It worker alright for me.

I have tried out xyz in many different cases and it looks really good. Congrats for your work.

However, I have faced  a small issue regarding a "white" space in my heatmap (as shown below).

I've checked and my .csv contains data for this region:

""""
-172.94,  -180.0,  97.342139310
-165.88,  -180.0,  93.901785182
...
-180.0 ,   -172.94,  101.030568117
-172.94,  -172.94,   97.920915039
-165.88,  -172.94,   93.564761222
...
(and more)"""

Did you ever come across any thing like this?

![Figure_1](https://user-images.githubusercontent.com/22683331/63041292-e4d7c680-be9d-11e9-8d0e-d4760fec6e3a.png)

![Figure_1-1](https://user-images.githubusercontent.com/22683331/63041451-4ef06b80-be9e-11e9-82f2-39913e376b6f.png)
